### ISSUE TITLE
Would be nice if globe color can be customized 

### ISSUE THREAD

**Issue by jarben** [ecomfe/echarts-gl#37]
Loading textures can take some time, there seems to be no way to configure globe's default color which shows before the texture is loaded:

![image](https://cloud.githubusercontent.com/assets/561365/25303916/9ddfb920-275c-11e7-9d92-e04e219ad04e.png)

It would be nice if it is possible to set color of the globe. 

I've found baseColor option but that seems to be changing the texture when it's loaded:

**Comment by pissang** [comment]
Yes base color will affect the texture color, which will simply multiply.

And for this case, maybe I should consider adding some extra option. Before that, you can load image by yourself and remove `baseColor` after `baseTexture` is set. GL support using HTMLImageElement as value of `baseTexture`.

**Comment by jarben** [comment]
Good point @pissang , will do! Globe color setting would be appreciated!
